recipes/echo/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "echo" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 50473083b662ec971aaf926f3b3ca703f416e2756522dd4c26da437f897ae0e1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - numpy
+    - qtpy
+    - setuptools
+
+test:
+  imports:
+    - echo
```

Comment:
it would probably be good to add to the imports tests `- echo.qt`

Reply:
Done